remo/profiles/admin.py

```diff
@@ -19,6 +20,58 @@
 admin.site.unregister(User)
 
 
+class RepProfileFilter(SimpleListFilter):
+    title = 'Rep profiles'
+    parameter_name = 'rep_profile'
+
+    def lookups(self, request, model_admin):
+        return (('False', 'No'), ('True', 'Yes'))
+
+    def queryset(self, request, queryset):
+        if self.value() == 'True':
+            return queryset.filter(groups__name='Rep')
+        return queryset
```

Comment:
I would prefer to exclude ``groups__name`` when ``self.value() == False``. This would make easier to apply multiple filters.

eg. select all reps that are not mentors